scripts/build/cc/100-gcc.sh

```diff
@@ -403,10 +403,12 @@ do_gcc_core_backend() {
         extra_config+=("--with-system-zlib")
     fi
 
-    # Some versions of gcc have a deffective --enable-multilib.
+    # Some versions of gcc have a defective --enable-multilib.
     # Since that's the default, only pass --disable-multilib.
     if [ "${CT_MULTILIB}" != "y" ]; then
         extra_config+=("--disable-multilib")
+    else
+        extra_config+=("--enable-multiarch")
```

Comment:
OK, so why do we have to --enable-multiarch when what we *do* are about
is multilib ?

Again, I fail to understand why --enable-multiarch is the solution to
enabling multilib...

I can see that we may want to add support for multiarch, but that's not
what this series is about, is it?